### PR TITLE
VIH-10457 INC5583659 Unable to book hearing when editing the date time before pressing book

### PR DESCRIPTION
**Pull request by marcogagliardi**
[VIH-10457](https://tools.hmcts.net/jira/browse/VIH-10457)

initialised court name and court code

### CODE REVIEW

**Review by will-craig — COMMENTED** [review]
Can we add a test too validate these properties are set when the component is initialised

**Comment by marcogagliardi** [comment]
test added

File: AdminWebsite/AdminWebsite/ClientApp/src/app/booking/hearing-schedule/hearing-schedule.component.spec.ts [hunk 1]
```diff
@@ -464,6 +467,13 @@ describe('HearingScheduleComponent returning to page', () => {
         expect(component.courtAddressControl.value).toBe(existingRequest.hearing_venue_id);
         expect(component.multiDaysHearing).toBe(existingRequest.isMultiDayEdit);
     });
+
+    it('should set controls for venue', () => {
+        component.ngOnInit();
+        expect(component.selectedCourtName).toBeTruthy();
+        expect(component.selectedCourtCode).toBeTruthy();
```
Comment (will-craig) [inline AdminWebsite/AdminWebsite/ClientApp/src/app/booking/hearing-schedule/hearing-schedule.component.spec.ts:474]: Is should assert the values, 'Bedford' and '333'

Comment (marcogagliardi) [reply AdminWebsite/AdminWebsite/ClientApp/src/app/booking/hearing-schedule/hearing-schedule.component.spec.ts:475]: test modified

**Review by will-craig — APPROVED** [review]
lg2m

**Review by komaldabhi1509 — APPROVED** [review]

**Review by komaldabhi1509 — APPROVED** [review]

**Pull request merged**